### DISCUSSION TITLE
Which raspi 4?

### DISCUSSION THREAD

**Discussion by nmz787** [SmoothWAN/SmoothWAN#23] [General & Help]
Is the 1GB RAM version acceptable? This one:
Raspberry Pi 4 Model B

**Comment by nmz787** [comment]
Actually it looks like this compute module is the thing in stock, is it any different in terms of chips and pins used? I don't care about the form factor or if I need to mess with JTAG, etc... https://www.digikey.com/en/products/detail/raspberry-pi/SC0326/13281653

**Reply by TalalMash** [reply]
This would work, but the Multi-WAN USB ports location will need to be changed for obvious reasons, you can skip using that and use the "Interfaces" menu. There are very few ports on that board which is a poor choice for a router.
All `Model B` are supported.
Safe minimum memory requirment for current SmoothWAN build is 512MB.

**Reply by bt61** [reply]
> Actually it looks like this compute module is the thing in stock, is it any different in terms of chips and pins used? I don't care about the form factor or if I need to mess with JTAG, etc... https://www.digikey.com/en/products/detail/raspberry-pi/SC0326/13281653

If you don't care about form factor, the Pi 400 keyboard with 4gb Pi4 built in seems like it's widely available in stock.  Shouldn't be any problem using that with SmoothWAN right @TalalMash ?

**Reply by TalalMash** [reply]
Yes! that didn't cross my mind. Port 3 or 4 in SmoothWAN should be ignored due to 3x USB ports vs 4x with Model B
If anyone managed to get it working, Ill update the image illustration to Pi 400 front ports.
